custom/icds_reports/urls.py

```diff
@@ -16,12 +16,15 @@
     DailyIndicators, InfantometerView, StadiometerView, MWCDDataView, IcdsDynamicMobileTemplateView,
     GovernanceHomeVisitAPI, GovernanceBeneficiaryAPI, GovernanceStateListAPI, GovernanceVHNDSAPI,
     MobileDashboardDownloadView, GovernanceCBEAPI, BiharDemographicsAPI, BiharMotherDetailsAPI, BiharVaccinesAPI,
-    BiharSchoolAPI, ServiceDeliveryDashboardDetailsView, PoshanProgressDashboardView, LSsLaunchedView
+    BiharSchoolAPI, ServiceDeliveryDashboardDetailsView, PoshanProgressDashboardView, LSsLaunchedView,
+    DownloadReleaseNotes, ReleaseNotesUpdateView, ReleaseDateView
 )
 
 
 dashboard_urls = [
     url('^mobile_app/$', MobileDashboardDownloadView.as_view(), name='icds_mobile_dashboard_download'),
+    url('^update_dashboard_release_notes/', ReleaseNotesUpdateView.as_view(), name=ReleaseNotesUpdateView.urlname),
```

Comment:
They do not need to be under dashboard urls

Reply:
Should I move them under url patterns?